lib/workers/bin/worker.rb

```diff
@@ -1,2 +1,10 @@
 type = ARGV.shift
+begin
+  # Remove the ::Runner class which is in all of them.
+  worker_name = type.gsub(/::Runner$/, '')
+  # Will set the process' name to eg. 'EVM Worker: MiqPriorityWorker'
+  # in ps aux, htop, ...
+  Process.setproctitle("EVM Worker: #{worker_name}")
```

Comment:
Some ideas that I think would be less than 3 lines to implement:

priority|event
---|---
MiqPriorityWorker 127|MiqEventHandler 124
MiqPriorityWorker 127 generic|MiqEventHandler 124 ems
MiqPriorityWorker 127 generic|MiqEventHandlerWorker 124 ems
MiqWorker: Priority 127|MiqWorker: EventHandler 124
Miq generic Worker: Priority 127|Miq ems Worker: EventHandler 124
Miq generic Priority Worker 127|Miq ems EventHandler Worker 124

Think I like the middle one. `MiqWorker: Priority 127` (I don't want the queue name, but if you want it, sounds good)

```ruby
short_name = name.gsub(/Miq/,'').gsub(/Worker/,'')
shortname = "" if short_name.downcase == queue
$0 = "Miq #{queue} #{short_name} Worker #{id}"
```